tests/unit/components/sl-date-picker-test.js

```diff
@@ -1,11 +1,19 @@
 import Ember from 'ember';
 import { moduleForComponent, test } from 'ember-qunit';
 import sinon from 'sinon';
+import ComponentInputId from 'sl-ember-components/mixins/sl-component-input-id';
 
 moduleForComponent( 'sl-date-picker', 'Unit | Component | sl date picker', {
     unit: true
 });
 
+test( 'Expected Mixins are present', function( assert ) {
+    assert.ok(
+        ComponentInputId.detect( this.subject() ),
+        'ComponentInputId Mixin is present'
```

Comment:
If going to reference the mixin name, would use the actual Mixin name, not the import alias.